categories.html

```diff
@@ -3,7 +3,7 @@
 layout: default
 permalink: /categories/
 ---
-{% include breadcrumbs.html %}
+{% include breadcrumbs.html parent="Categories" %}
```

Comment:
I don't think it makes sense for this page to specify Categories as a parent since this page is itself Categories. The breadcrumb navigates to itself.